contracts/SpeedMarkets/SpeedMarketsAMM.sol

```diff
@@ -520,43 +530,14 @@ contract SpeedMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReent
             !SpeedMarket(market).resolved();
     }
 
-    /// @notice return all market data for an array of markets
-    function getMarketsData(address[] calldata marketsArray) external view returns (MarketData[] memory) {
-        MarketData[] memory markets = new MarketData[](marketsArray.length);
-        for (uint i = 0; i < marketsArray.length; i++) {
-            SpeedMarket market = SpeedMarket(marketsArray[i]);
-            markets[i].user = market.user();
-            markets[i].asset = market.asset();
-            markets[i].strikeTime = market.strikeTime();
-            markets[i].strikePrice = market.strikePrice();
-            markets[i].direction = market.direction();
-            markets[i].buyinAmount = market.buyinAmount();
-            markets[i].resolved = market.resolved();
-            markets[i].finalPrice = market.finalPrice();
-            markets[i].result = market.result();
-            markets[i].isUserWinner = market.isUserWinner();
-            if (marketHasCreatedAtAttribute[marketsArray[i]]) {
-                markets[i].createdAt = market.createdAt();
-            }
-        }
-        return markets;
+    /// @notice
+    function getTimeThresholdsForFees() external view returns (uint[] memory) {
```

Comment:
the variable is already public, maybe best to move this to data contract

Reply:
Will move to Data